src/frontends/pytorch/src/frontend.cpp

```diff
@@ -60,17 +68,51 @@ FrontEnd::FrontEnd() : m_op_translators(get_supported_ops()) {}
 
 std::shared_ptr<Model> FrontEnd::convert(const InputModel::Ptr& model) const {
     auto converted_model = convert_partially(model);
-    normalize(converted_model);
-    std::set<std::string> unconverted_ops_types = get_unconverted_types_from_model(converted_model);
-    std::stringstream ops_str;
-    for (auto&& op_type : unconverted_ops_types) {
+    std::stringstream error_msg("Model wasn't fully converted.");
+    error_msg << "Model wasn't fully converted.";
+
+    std::string norm_err;
+    try {
+        normalize(converted_model);
+    } catch (const std::exception& e) {
+        norm_err = "\n-- normalize step failed with: " + std::string(e.what());
```

Comment:
It doesn't contain information about which particular transformation fails -- it's hard to follow -- I'm looking at an example of error log that you sent me earlier. Can we capture the name of transformation that failed?

Reply:
Exactly, please create a ticket against core to add such information?

Reply:
> Exactly, please create a ticket against core to add such information?

Done, ticket 113306